src/envoy/mixer/http_control.cc

```diff
@@ -210,11 +210,19 @@ void HttpControl::Check(HttpRequestDataPtr request_data, HeaderMap& headers,
   SetStringAttribute(kOriginUser, origin_user, &request_data->attributes);
   log().debug("Send Check: {}", request_data->attributes.DebugString());
 
-  auto check_on_done = [this, on_done](const Status& status) {
+  auto check_on_done = [this, on_done, request_data](const Status& status) {
     if (status.ok()) {
       if (!quota_attributes_.attributes.empty()) {
-        log().debug("Send Quota: {}", quota_attributes_.DebugString());
-        mixer_client_->Quota(quota_attributes_, on_done);
+        for (const auto& attribute : quota_attributes_.attributes) {
+          request_data->attributes.attributes[attribute.first] =
```

Comment:
this request_data->attributes is send to Report call too.  Do we want to add quota metrics in the report call.  If so,  we are fine.  Otherwise, we may want to clone a new attribute for Quota Call.